firefox-ios/Client/ContentBlocker/TabContentBlocker.swift

```diff
@@ -20,15 +20,14 @@ protocol ContentBlockerTab: AnyObject {
 }
 
 @MainActor
-class TabContentBlocker {
+class TabContentBlocker: Notifiable {
     weak var tab: ContentBlockerTab?
     let logger: Logger
 
     var isEnabled: Bool {
         return false
     }
 
-    @objc
     nonisolated func notifiedTabSetupRequired() {}
```

Comment:
I think we can remove the `nonisolated` here, and make the same adjustment in the overriden function in `FirefoxTabContentBlocker`, which will enable us to remove the extra `ensureMainThread` over there (since we have the `ensureMainThread` under the `handleNotifications` now)